src/gluonts/model/simple_feedforward/_network.py

```diff
@@ -92,34 +90,31 @@ def __init__(
             )
             self.scaler = MeanScaler() if mean_scaling else NOPScaler()
 
-    def get_distr(self, F, past_target: Tensor) -> Distribution:
+    def get_distr_args(self, F, past_target: Tensor):
```

Comment:
This returns a `Tuple` of three `Tensor`: can you add the output type annotation? `Tuple[Tensor, Tensor, Tensor]`

Reply:
Yes, I have fixed that.